spec/models/game_spec.rb

```diff
@@ -63,4 +63,12 @@
       end
     end
   end
+
+ describe "#populate_board" do
+    it "should populate board only if both black and white player ids are present" do
+      game = Game.create(white_player_id: 0, black_player_id: 1)
```

Comment:
Thanks Jacob.  Jim helped me get to the user creation part, but I couldnt quite get it to work until the above.  We were so close.  Got it working now.  Will send up again for review.